src/native/src/3rdparty/glad/include/KHR/khrplatform.h

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
It can also be a good idea to regenerate those types of things often, as they generally update the opengl xml spec quite a bit with bug fixes and new extensions and stuff.